### DISCUSSION TITLE
Get the line number on which there is an error

### DISCUSSION THREAD

**Discussion by MuhammadHaseebAli** [json-schema-org/community#53] [Q&A]
Is it possible to get the line number on which there is an error while validation the schema definition
for example, if we have a schema and we make a typo like "objects" (the correct one is "object") and it will tell us that there is an error on line number "20"

**Comment by Relequestual** [comment]
Some tooling will give you the JSON Pointer, such as https://github.com/hyperjump-io/json-schema-validator
You can then use the JSON Pointer to determine the line using another tool, such as https://github.com/epoberezkin/json-source-map

I haven't done this myself, but I believe it's possible.

**Comment by karenetheridge** [comment]
This was also asked on https://stackoverflow.com/questions/69255306/get-the-line-number-on-which-there-is-an-error-in-json-schema-definition.
